.github/workflows/beam_PreCommit_Java_Flink.yml

```diff
@@ -0,0 +1,94 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Java Flink Runner PreCommit
+
+on:
+  push:
+    tags: ['v*']
+    branches: ['master', 'release-*']
+    paths:
+      - 'model/**'
+      - 'runners/flink/**'
+      - 'release/**'
+      - '.github/workflows/beam_PreCommit_Java_Flink.yml'
+  pull_request_target:
+    branches: ['master', 'release-*']
+    paths:
+      - 'model/**'
+      - 'runners/flink/**'
+      - 'release/**'
+  issue_comment:
+    types: [created]
+  schedule:
+  - cron: '20 */6 * * *'
```

Comment:
What is the intention to run the workflow exactly at the 20th minute?

Reply:
manually add some randomness to distribute the job, then not all cron job start at 0th minute